### ISSUE TITLE
BOINC 7.16.11 stops "turn screen off in N minutes" in Windows 10

### ISSUE THREAD

**Issue by Ageless93** [BOINC/boinc#4018]
I noticed this the other days when I allowed BOINC to run some WCG work. I left my computer and went and did the dishes. Coming back half an hour later both my monitors were still on. I've got them set both in Windows, to turn off after 10 minutes. 

So today I tested this again. 
1. Started BOINC on my secondary monitor. 
2. Allowed WCG to fill the cache with OpenPandemics and let it run. After 15 minutes the monitors were still on. 
3. I then allowed Universe@Home to get tasks, paused WCG and let Universe run. 19 minutes later the monitors were still on. 
4. Then I paused everything (by project) but left BOINC open. After 17 minutes both monitors are still on. 
5. I then minimized BOINC to the taskbar. 18 minutes later, monitors still on. 
6. Then I exited BOINC and after waiting 10 minutes the monitors go off. 

So what in BOINC has changed that it stops my monitors from going to sleep?

**Comment by AenBleidd** [comment]
@Ageless93, I have similar behavior even on BOINC 7.16.7. Not sure it's related to BOINC because I have it running always in background (paused when in use). In my case Windows doesn't lock my laptop after timeout (not always but quite often).
Also I'm not sure your case and my case have similar reason

**Comment by Ageless93** [comment]
Well, that's why I tested it with BOINC running, even with it minimized. 
If my monitors react normally when BOINC isn't running, but they won't if BOINC is running, then... it's Windows that does this?

**Comment by RichardHaselgrove** [comment]
I usually turn my main monitors off with the power switches, which doesn't help in this case.

But at the moment I'm travelling with a Windows 10 laptop. At night, I set it to turn the screen off - BOINC 7.16.11 runs (minimised), all my browser tabs are active but hidden behind the lock screen. Works as advertised - room goes dark and stays dark.

**Comment by AwfulCharlie** [comment]
I've never had this issue, including now with clients 7.16.11 & 7.16.7

That said, I don't use *any* screensavers from Boinc or the attached projects (just use windows screensaver 'Blank' after 14 mins and Screen off after 15 mins)

Maybe it is a Boinc screensaver that is holding your monitors up?

**Comment by Ageless93** [comment]
![No BOINC Screen Saver](https://i.ibb.co/Z6tFdZJ/no-screen-saver.jpg)

Like I said in my original post: "I don't use the BOINC screen saver, nor do I allow the projects to show graphics (where applicable)"

**Comment by psypherium** [comment]
This issue is not exclusive to Windows, it is also a problem in Arch Linux, and probably other Linuxes, and my guess would be that it also happens on mac. Boinc version 7.18.1.

**Comment by Vulpine05** [comment]
After a quick search online, apparently Windows has a power troubleshooter.  Settings-> Update & Security -> Troubleshoot, then click on "Additional troubleshooters".  There is one for Power.  @Ageless93 , could you try that if you haven't already?  I know this doesn't help with Linux.

I haven't had this problem on the two desktops and two laptops I've used BOINC (Windows 10 and Ubuntu 18.04/20.04), so I'm not sure how to replicate this.

Just curious, is anyone experiencing this problem using multiple monitors?

**Comment by psypherium** [comment]
I am experiencing this problem using multiple monitors. You may be on to something.

**Comment by Vulpine05** [comment]
Well, I do have a second monitor for my work from home setup.  Let me find a DisplayPort cable or adapter and I will be able to see if I can replicate this bug.  Give me a few days and I will let you know.

**Comment by Vulpine05** [comment]
Apparently my 1660 Ti has two HDMI ports!  I plugged in the second monitor, waited for my screen to turn off, and it did.  I rebooted, loaded BOINC again, moved the window to display 2 (the new monitor), waited, and the screen turned off.  Sorry, I can't replicate this bug.  I'm using Windows 10 and BOINC 7.20.2.

Windows is set to turn off the display after 3 minutes, no screen saver is used.  Both monitors are at 1080, displays are not duplicated.

**Comment by psypherium** [comment]
BOINC was active processing tasks during the testing?

**Comment by Vulpine05** [comment]
Yes, tasks were running.  I may test this again another day, I am curious if any different power settings may affect this.  Although, that wouldn't necessarily explain how this occurs in Linux.
